engine/runtime/src/main/scala/org/enso/interpreter/runtime/IrToTruffle.scala

```diff
@@ -539,17 +539,40 @@ class IrToTruffle(
                     effectContext,
                     true
                   )
-                val rootNode = MethodRootNode.build(
-                  language,
-                  expressionProcessor.scope,
-                  moduleScope,
-                  () => bodyBuilder.bodyNode(),
-                  makeSection(moduleScope, methodDef.location),
-                  cons,
-                  methodDef.methodName.name
-                )
+
+                val operators = ".!$%&*+-/<>?^~\\"
+                def isOperator(n: Name): Boolean = {
+                  n.name
+                    .chars()
+                    .allMatch(operators.indexOf(_) >= 0)
+                }
```

Comment:
Later I will need to be able to check `isOperator` also in the compiler at the type inference pass, so this should probably be somewhere reachable from both places.

But I'm happy to move it when the time comes, it seems OK to be here for now

Reply:
I wasn't sure what is the right place for such method.

Reply:
Another incarnation of this `isOperator` check: https://github.com/enso-org/enso/pull/9528/commits/9d7c75063fc98f8b059d1ecbd38a2547cae31d0d#r1570045334 CCing @kazcw  and @4e6.